### PR TITLE
Add Robotframework image for production test

### PR DESCRIPTION
**Pull request by RikardO-HM**

### CODE REVIEW

**Comment by mattiasbus** [comment]
* It seems that this is not rebased on the top of master?
*  I created another branch with this done(https://github.com/hostmobility/meta-mobility-poky-distro/tree/mb/rebased-robotpackagegroup) 
* Did a Jenkins build in http://anansi.setekgroup.local:8080/job/Testing-Mattias/job/robot-framework-image/

File: recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_can.network [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Match]
+Name=can*
+
+[CAN]
+BitRate=500K
+RestartSec=100ms
```
Comment (mattiasbus) [inline recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_can.network:6]: * we can have a longer RestartSec here to make it easier to detect errorframes
* we can enable canfd here as well with 
```
[CAN]
DataBitRate=4000000
FDMode=True
```
* https://pengutronix.de/en/blog/2022-02-04-initializing-can-interfaces-with-systemd-networkd.html

File: recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_unit.bash
Comment (mattiasbus) [inline recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_unit.bash:1]: this should be taken care of by the ref_can.network

File: recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_unit_setup.bash [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+cupsaccept ZebraGX430t
+cupsenable ZebraGX430t
+
+
+
+# configuration is done stop this service and disable it we dont need it any more.
+systemctl disable ref_unit_setup.service
+systemctl stop ref_unit_setup.service
+
+echo done, reboot the system!
```
Comment (mattiasbus) [inline recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_unit_setup.bash:92]:
```suggestion
echo "Done, reboot the system!"
```

File: recipes-core/fs-init/files/mxv-base/resizefs.service [hunk 3]
```diff
@@ -3,7 +3,7 @@ Description=start resize script to increase rootfs to its partition full size
 
 [Service]
 Type=simple
-ExecStart=/bin/sh -c 'sleep 5 ; /usr/sbin/resizefs.sh mmcblk2 2'
+ExecStart=/bin/sh -c 'sleep 5 ; /usr/sbin/resizefs.sh mmcblk2 1'
```
[the remark this answers is not in the document]
Comment (RikardO-HM) [reply recipes-core/fs-init/files/mxv-base/resizefs.service]: dropped commit

**Pull request merged**